WordPress/Classes/Models/StatsRecord+CoreDataClass.swift

```diff
@@ -42,7 +43,9 @@ public enum StatsRecordType: Int16 {
         case .lastPostInsight, .allTimeStatsInsight, .publicizeConnection, .followers:
 
             return false
-        case .postStats, .blogStats:
+        case .postStats,
```

Comment:
Can we modify this for consistency with line 43 (i.e., all cases on a single line)?

Reply:
It was actually an artifact of this not being updated from the upstream branch, it's split across multiple lines for the other case too after merging it back in :)